CppSamples/ChilliConnectCppDemo/HttpSystem.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#pragma comment(lib, "winhttp.lib")
+
+#include <string>
+#include <Windows.h>
+#include <map>
+#include <winhttp.h>
+
+using std::string;
```

Comment:
Not really that important in this case but it is against C++ best practices to use "using namespace" in a header file as it will then be enforced in any file that includes this header